src/components/Button/index.js

```diff
@@ -0,0 +1,7 @@
+import Button from './Button';
+import _IconButton from './IconButton';
+import _LoginButton from './LoginButton';
+
+export default Button;
+export const IconButton = _IconButton;
+export const LoginButton = _LoginButton;
```

Comment:
This can be simplified to `export { default as LoginButton } from './LoginButton`.